fenix/app/src/main/java/org/mozilla/fenix/home/sessioncontrol/viewholders/PrivateBrowsingDescriptionViewHolder.kt

```diff
@@ -51,9 +59,17 @@ class PrivateBrowsingDescriptionViewHolder(
 
     @Composable
     override fun Content() {
-        PrivateBrowsingDescription(
-            onLearnMoreClick = interactor::onLearnMoreClicked,
-        )
+        val settings = composeView.context.settings()
+
+        if (settings.feltPrivateBrowsingEnabled) {
```

Comment:
Ah, I see our UI tests are also failing because this change. We will need to add a feature flag in our UI tests.

See: 
https://searchfox.org/mozilla-mobile/source/firefox-android/fenix/app/src/androidTest/java/org/mozilla/fenix/helpers/FeatureSettingsHelperDelegate.kt
https://searchfox.org/mozilla-mobile/source/firefox-android/fenix/app/src/androidTest/java/org/mozilla/fenix/helpers/HomeActivityTestRule.kt#58

We want to tell the UI tests that `settings.feltPrivateBrowsingEnabled` is false so we can keep the UI tests running in the happy state.

Given this large change and the new comment about strings from flod, I am gonna set this to "changes needed" so that we can do another review of this.

Reply:
After a conversation with @boek we will be defaulting this flag to false for the time being while this feature is still being worked on.
This will clear the UI test failures and we can move forward with this PR after I push the other changes.

Reply:
Perfect!